web_src/js/components/RepoActionView.vue

```diff
@@ -393,7 +413,7 @@ export function initRepositoryActionView() {
         <button class="ui basic small compact button red" @click="cancelRun()" v-else-if="run.canCancel">
           {{ locale.cancel }}
         </button>
-        <button class="ui basic small compact button tw-mr-0 tw-whitespace-nowrap link-action" :data-url="`${run.link}/rerun`" v-else-if="run.canRerun">
+        <button class="ui basic small compact button link-action" :data-url="`${run.link}/rerun`" v-else-if="run.canRerun">
```

Comment:
- `tw-mr-0` should eliminate a 4px margin-right caused by button CSS, can we restore it.
- `tw-whitespace-nowrap` should prevent the button text from ever wrapping, like on small viewport. Did you check?

Reply:
Ah I see you put CSS below.

Reply:
These buttons are all in `if-else`

The `tw-mr-0` you added was only on the last `else` button.